tests/services/schemas/test_affiliation.py

```diff
@@ -14,98 +14,16 @@
 
 
 def test_valid():
-    valid_full = {
-        "name": "Entity One",
-        "identifiers": [{
-            "identifier": "03yrm5c26",
-            "scheme": "ror"
-        }]
-    }
-    assert valid_full == AffiliationSchema().load(valid_full)
-
-
-def test_valid_no_identifiers():
-    valid_affiliation = {
-        "name": "Entity One",
-    }
-    assert valid_affiliation == AffiliationSchema().load(valid_affiliation)
-
-
-def test_valid_empty_identifiers():
     valid_affiliation = {
+        "id": "test",
         "name": "Entity One",
-        "identifiers": []
     }
     assert valid_affiliation == AffiliationSchema().load(valid_affiliation)
 
 
-def test_valid_empty_scheme():
-    valid_affiliation = {
-        "name": "Entity One",
-        "identifiers": [{
-            "identifier": "03yrm5c26",
-            "scheme": ""
-        }]
-    }
-
-    loaded = AffiliationSchema().load(valid_affiliation)
-    valid_affiliation["identifiers"][0]["scheme"] = "ror"
-    assert valid_affiliation == loaded
-
-
-def test_invalid_no_name():
-    invalid_no_name = {
-        "identifiers": [{
-            "identifier": "03yrm5c26",
-            "scheme": "ror"
-        }]
+def test_invalid_no_id():
```

Comment:
This will be valid, but dealt with in a separate PR, right?

Reply:
Youp, is fixed in the other open PR already, didnt rebase it yet 😅